src/client/components/pages/parts/search-results.js

```diff
@@ -21,122 +21,257 @@ import * as bootstrap from 'react-bootstrap';
 
-		return (
-			<tr key={result.bbid}>
-				{
-					!props.condensed &&
+			const secondaryAliases = !this.props.condensed && aliases &&
+				_differenceBy(aliases, [result.defaultAlias], 'id').map(alias => alias.name).join(', ');
+
+			const disambiguation = result.disambiguation ? <small>({result.disambiguation.comment})</small> : '';
+			// No redirect link for Area entity results
+			const link = result.type === 'Area' ?
+				`//musicbrainz.org/area/${result.bbid}` :
+				`/${_kebabCase(result.type)}/${result.bbid}`;
+
+			/* eslint-disable react/jsx-no-bind */
+			return (
+				<tr key={result.bbid}>
+					{
+						!this.props.condensed &&
+						<td>
+							{
+								this.props.user ?
+									<input
+										checked={this.state.selected.find(selected => selected.bbid === result.bbid)}
+										className="checkboxes"
+										type="checkbox"
+										onChange={() => this.toggleRow(result)}
+									/> : null
+							}
+							{genEntityIconHTMLElement(result.type)}{_startCase(result.type)}
+						</td>
+					}
 					<td>
-						{genEntityIconHTMLElement(result.type)}{_startCase(result.type)}
+						<a href={link}>
+							{name} {disambiguation}
+						</a>
 					</td>
+					{
+						!this.props.condensed &&
+						<td>
+							{secondaryAliases}
+						</td>
+					}
+				</tr>
+			);
+		});
+		let tableCssClasses = 'table table-striped';
+		if (this.props.condensed) {
+			tableCssClasses += ' table-condensed';
+		}
+
+		return (
+			<div>
+				{
+					this.props.user ?
+						<div>
+							<AddToCollectionModal
+								bbids={this.state.selected.map(selected => selected.bbid)}
+								closeModalAndShowMessage={this.closeModalAndShowMessage}
+								entityType={this.state.selected[0]?.type}
+								handleCloseModal={this.onCloseModal}
+								show={this.state.showModal}
+								userId={this.props.user.id}
+							/>
+						</div> : null
 				}
-				<td>
-					<a href={link}>
-						{name} {disambiguation}
-					</a>
-				</td>
 				{
-					!props.condensed &&
-					<td>
-						{secondaryAliases}
-					</td>
+					!this.props.condensed &&
+					<h3 style={{color: '#754e37'}}>
+						Search Results
+					</h3>
 				}
-			</tr>
-		);
-	});
-	let tableCssClasses = 'table table-striped';
-	if (props.condensed) {
-		tableCssClasses += ' table-condensed';
-	}
-	return (
-		<div>
-			{
-				!props.condensed &&
-				<h3 style={{color: '#754e37'}}>
-					Search Results
-				</h3>
-			}
-			<hr className="thin"/>
-			<Table
-				responsive
-				className={tableCssClasses}
-			>
+				<hr className="thin"/>
+				<Table
+					responsive
+					className={tableCssClasses}
+				>
+					{
+						!this.props.condensed &&
+						<thead>
+							<tr>
+								<th className="col-sm-3">Type</th>
+								<th className="col-sm-5">Name</th>
+								<th className="col-sm-4">Aliases</th>
+							</tr>
+						</thead>
+					}
+					<tbody>
+						{results}
+					</tbody>
+					{
```

Comment:
This whole block should be after the closing `</Table>` tag.

You'll be able to get rid of the whole `className="pull-left margin-top-d5"` on the button too.


I tried with an `<hr className='thin'/>` in-between the table and the button, too. I'll let you decide which one looks best in your opinion:

![FireShot Capture 101 - Search Results – BookBrainz - localhost](https://user-images.githubusercontent.com/6179856/88653020-d1803400-d0cb-11ea-8d57-b16862eeee64.png)
![FireShot Capture 102 - Search Results – BookBrainz - localhost](https://user-images.githubusercontent.com/6179856/88653034-d47b2480-d0cb-11ea-9bf7-1dbc3f0723c7.png)